src/Player.ts

```diff
@@ -13,15 +13,23 @@ function writeSync(term: xterm.Terminal, data: string) {
   }
 }
 
-function createTerminal(options?: xterm.ITerminalOptions): xterm.Terminal {
+function createTerminal(options: IPlayerOptions): xterm.Terminal {
+  const opts: xterm.ITerminalOptions = {
+    fontSize: options.fontSize || 15,
+    fontFamily: options.fontFamily || 'Consolas, Menlo',
+    fontWeight: options.fontWeight || 'normal',
+    fontWeightBold: options.fontWeightBold || 'bold',
+    theme: options.theme || {},
```

Comment:
I think u can just remove this line `theme: options.theme || {},`

Reply:
We need this line to set the theme, do you mean remove the default value?

Reply:
I misread